pages/content-guidelines.md

```diff
@@ -0,0 +1,104 @@
+---
+permalink: /content-guidelines/
+title: Writing a great post
+---
+
+Keep it user centered
+---------------------
+
+Blog posts should offer something to the reader, whether that’s a new
+tool for them to use, a guide they can use to improve their own work, or
+educational information about a complicated topic or process. You should
+begin your post with what’s new and how your readers can use this new
+thing.
```

Comment:
> begin your post with what’s new and how your readers can use this new thing.
I see what we're getting at here but maybe we rephrase to something like: begin your post with the thing your readers should take away from your post, and why it matters to them. Concerned people might be confused that even if their post is education it has to begin with a new thing people can use.